File: woodroid/src/main/java/me/gilo/woodroid/data/callbacks/ReviewsCallback.java
package me.gilo.woodroid.data.callbacks;


import com.google.gson.annotations.SerializedName;
import me.gilo.woodroid.models.ProductReview;

import java.util.ArrayList;



public class ReviewsCallback {
    @SerializedName("product_reviews")
    ArrayList<ProductReview> productReviews;

    public ArrayList<ProductReview> getProductReviews() {
        return productReviews;
    }

    public void setProductReviews(ArrayList<ProductReview> productReviews) {
        this.productReviews = productReviews;
    }
}

File: woodroid/src/main/java/me/gilo/woodroid/data/callbacks/ReviewsCallback.kt
package me.gilo.woodroid.data.callbacks


import com.google.gson.annotations.SerializedName
import me.gilo.woodroid.models.ProductReview

import java.util.ArrayList


class ReviewsCallback {
    @SerializedName("product_reviews")
    lateinit var productReviews: ArrayList<ProductReview>
}
